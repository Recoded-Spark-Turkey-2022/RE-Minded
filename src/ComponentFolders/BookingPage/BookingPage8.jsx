import React from 'react';
import Card from './Card';

function BookingPage8() {
  return (
    <div className="flex flex-col  p-24">
      <Card
        header="YOUR REQUEST HAS BEEN SUBMITTED!"
        content="You will receive an email guiding you to book a date and time soon."
      />
      <div className="font-poppins shadow-zinc-300 flex flex-col items-center my-16 mx-auto px-16 py-24 shadow-md">
        <h1 className="text-xl pb-12 md:text-2xl lg:text-3xl lg:flex justify-center">
          Request Submitted
        </h1>
        <h2 className="text-md pb-10 md:text-lg lg:text-xl lg:flex justify-center">
          You Will Receive A Confirmation Email Soon Please Keep An Eye On Your
          Mail.
        </h2>
        <div className="flex justify-center">
          <button
            type="button"
            className="m-6 rounded-md box-border text-md md:text-lg  pr-6 pl-6 pt-2 pb-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 lg:text-2xl"
          >
            BACK TO HOME
          </button>
        </div>
      </div>
    </div>
  );
}

export default BookingPage8;
