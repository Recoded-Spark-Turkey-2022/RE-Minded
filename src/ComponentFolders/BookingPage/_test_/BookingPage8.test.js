import React from 'react';
import renderer from 'react-test-renderer';
import BookingPage8 from '../BookingPage8';

test('Booking8 Page snapshot test', () => {
  const component = renderer.create(<BookingPage8 />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
