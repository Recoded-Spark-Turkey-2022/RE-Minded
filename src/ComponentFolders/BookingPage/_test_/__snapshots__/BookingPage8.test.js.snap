// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking1 Page snapshot test 1`] = `
<div
  className="flex flex-col items-center p-12"
>
  <div
    className=" flex flex-col p-14 font-poppins "
  >
    <h1
      className="text-2xl pb-6 md:text-2xl lg:text-4xl"
    >
      LET'S MATCH YOU WITH THE RIGHT THERAPIST
    </h1>
    <p
      className="text-lg  md:text-xl opacity-50 lg:text-2xl lg:pr-40  "
    >
      Please fill out this short questionnaire to provide some general and anonymous background about you and the issues you'd like to deal with in online therapy. It would help us match you with the most suitable therapist for you. 
    </p>
  </div>
  <div
    className="font-poppins shadow-zinc-300 flex flex-col w-full md:max-w-2xl lg:max-w-4xl my-16 mx-auto px-8 py-4 shadow-md"
  >
    <div
      className="p-10"
    >
      <h1
        className="text-xl pb-6 md:text-2xl lg:text-3xl lg:flex justify-center mt-4"
      >
        What type of counseling are you looking for?
      </h1>
      <div
        className="flex flex-col pl-6 text-md md:text-lg lg:text-xl"
      >
        <div
          className="shadow-zinc-300 flex flex-col w-full mb-6 mt-12 px-4 py-4 shadow-md hover:bg-cyan-400 cursor-pointer"
        >
          Individual counseling
        </div>
        <div
          className="shadow-zinc-300 flex flex-col w-full mb-6 px-4 py-4 shadow-md hover:bg-cyan-400 cursor-pointer"
        >
          Teen counseling (for my child)
        </div>
      </div>
    </div>
    <div
      className="flex justify-center md:justify-start lg:justify-start "
    >
      <button
        className="m-6 rounded-md box-border text-xl pr-6 pl-6 pt-2 pb-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 lg:text-2xl"
        type="button"
      >
        NEXT
      </button>
    </div>
  </div>
</div>
`;

exports[`Booking8 Page snapshot test 1`] = `
<div
  className="flex flex-col  p-24"
>
  <div
    className=" flex flex-col p-14 font-poppins "
  >
    <h1
      className="text-2xl pb-6 md:text-2xl lg:text-4xl"
    >
      YOUR REQUEST HAS BEEN SUBMITTED!
    </h1>
    <p
      className="text-lg  md:text-xl opacity-50 lg:text-2xl lg:pr-40  "
    >
      You will receive an email guiding you to book a date and time soon.
    </p>
  </div>
  <div
    className="font-poppins shadow-zinc-300 flex flex-col items-center my-16 mx-auto px-16 py-24 shadow-md"
  >
    <h1
      className="text-xl pb-12 md:text-2xl lg:text-3xl lg:flex justify-center"
    >
      Request Submitted
    </h1>
    <h2
      className="text-md pb-10 md:text-lg lg:text-xl lg:flex justify-center"
    >
      You Will Receive A Confirmation Email Soon Please Keep An Eye On Your Mail.
    </h2>
    <div
      className="flex justify-center"
    >
      <button
        className="m-6 rounded-md box-border text-md md:text-lg  pr-6 pl-6 pt-2 pb-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 lg:text-2xl"
        type="button"
      >
        BACK TO HOME
      </button>
    </div>
  </div>
</div>
`;
